hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMMetadataStoreImpl.java

```diff
@@ -190,6 +200,9 @@ public void start(OzoneConfiguration config)
 
       checkTableStatus(statefulServiceConfigTable,
           STATEFUL_SERVICE_CONFIG.getName());
+
+      mxBean = MBeans.register("SCMMetadataStore", "SCMMetadataStoreImpl",
```

Comment:
I suspect this needs an "unRegister" in the `stop()` method?

Reply:
@sodonnel Thank you for your reminder, updated the PR, please have a look.